lib/oneview-sdk/resource/api200/logical_interconnect_group.rb

```diff
@@ -53,14 +47,25 @@ def self.get_default_settings(client)
       # @param [String] type Interconnect type
       # @raise [StandardError] if a invalid type is given then raises an error
       def add_interconnect(bay, type)
+        interconnect_type = OneviewSDK::Interconnect.get_type(@client, type)
+        raise OneviewSDK::NotFound unless interconnect_type
```

Comment:
Provide a message for the user saying the resource type and name that wasn't found